lib/listener.ts

```diff
@@ -0,0 +1,13 @@
+import { Config } from './config';
+import { MissingAttributeError } from './utils/errors';
+
+export class Listener {
+	public static on(eventName: string, callback: Function) {
+		if (eventName.length <= 0) throw new MissingAttributeError('eventName');
```

Comment:
the right param in error adapter is 'eventName' as string or eventName as it comes from function param?

Reply:
should be "Event name" to avoid confusion with param name